app/views/companies/show.html.erb

```diff
@@ -23,6 +23,11 @@
             <li style="display: inline"><%= service.name %></li>
           <% end %>
         </ul>
+        <% if current_admin %>
+          <a href=<%= "/companies/#{@company.id}/edit"%>>
```

Comment:
Use the link_to helper and use `edit_company_path(@company)` instead of hard-coding the href. No need to add a `<button>` tag, you can just add those classes to the link_to and it will render the same thing @acolletti21